resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,68 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="panel panel-default">
+                <div class="panel-heading">Login</div>
+                <div class="panel-body">
+                    <form class="form-horizontal" role="form" method="POST" action="{{ route('login', ['tenant' => $tenant]) }}">
```

Comment:
```php
{{ route('login', ['tenant' => $tenant]) }}
```
everywhere where we use `route('login')`we have to pass $tenant. Is there better solution?